word_language_model/main.py

```diff
@@ -92,7 +103,12 @@ def batchify(data, bsz):
 ###############################################################################
 
 ntokens = len(corpus.dictionary)
-model = model.RNNModel(args.model, ntokens, args.emsize, args.nhid, args.nlayers, args.dropout, args.tied).to(device)
+if args.model == 'Transformer':
+	model = model.TransformerSeq2Seq(ntokens, ntokens, args.emsize, args.transformer_head,
+                                     args.nlayers, args.nlayers, args.nhid, 
+                                     args.dropout).to(device) 
+else:
+	model = model.RNNModel(args.model, ntokens, args.emsize, args.nhid, args.nlayers, args.dropout, args.tied).to(device)
```

Comment:
Now that the arguments largely match up, can we have a "SeqModel" (or such) class that has a field to which it assigns either the RNN or Transformer based on the value of "args.model"?

Reply:
let's discuss this and make a decision.